### ISSUE TITLE
Can't sign in due to plz--response error

### ISSUE THREAD

**Issue by progfolio** [alphapapa/ement.el#118]
Can no longer log in due to https://github.com/alphapapa/plz.el/issues/12.
I get the following error:

```
⛔ Warning (emacs): Ement: ‘plz’ request for .well-known URI signaled an error: (plz-http-error "plz--response: Unable to parse HTTP response status line" "HTTP/2 200 
")
```

Adjusting the regexp mentioned in that issue works for me.

**Comment by alphapapa** [comment]
I'm trying to extend the `plz` test suite to test with HTTP 2.0 so I can solve that problem comprehensively.

In the meantime, I'd suggest installing the latest stable release of `plz` from ELPA, which doesn't have that change.

**Comment by progfolio** [comment]
> I'd suggest installing the latest stable release of plz from ELPA, which doesn't have that change.

Will do! Thank you.

**Comment by alphapapa** [comment]
This should now be fixed in v0.3 of `plz`.  Please let me know if it works for you.  Thanks.
